src/routes/github/create-branch/github-create-branch-get.test.ts

```diff
@@ -3,6 +3,7 @@ import supertest from "supertest";
 import { getLogger } from "config/logger";
 import { getFrontendApp } from "~/src/app";
 import { getSignedCookieHeader } from "test/utils/cookies";
+import { GetRepositoriesQuery } from "~/src/github/client/github-queries";
```

Comment:
if you slap 
```
defaultBranchRef {
            name
          }
```

into the `GetRepositoriesQuery` we can get the default branch name and pass that through as the first selection in the branch list.

Can do this in seperate PR if you wanna.

Reply:
in fact, there is a story [ARC-1608](https://softwareteams.atlassian.net/browse/ARC-1608) which we can cover it in

Reply:
I tried using `defaultBranchRef`, it just returns null and a Forbidden error. Lets just do it separate in Arc-1608